pymc_workflow_analyzer/parser.py

```diff
@@ -0,0 +1,70 @@
+import ast
+from pymc.distributions import __all__ as pymc_distributions
```

Comment:
Is there any better way to get all the distributions present in PyMC?

Reply:
I think this will miss distributions from other modules (e.g. pymc-experimental), GP stuff, the Simulator for approximate likelihoods

Reply:
Could you suggest a better approach for listing all the distributions, models, samplers, etc.?